#!/usr/bin/env python
PACKAGE = "velodyne_pointcloud"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_range", double_t, 0, "min range to publish", 0.0, 0.9, 10.0)
gen.add("max_range", double_t, 0, "max range to publish", 130, 0.9, 200)
gen.add("view_direction", double_t, 0, "angle defining the center of view", 0.0, 0.0, 6.283185)
gen.add("view_width", double_t, 0, "angle defining the view width", 6.283185, 0.0, 6.283185)

exit(gen.generate(PACKAGE, "velodyne_pointcloud", "VelodyneConfig"))
